README.md

```diff
@@ -1,6 +1,8 @@
-# DEVELOPER PREVIEW
+# AWS CloudFormation Resource Provider Python Plugin
 
-We're excited to share our progress with adding new languages to the CloudFormation CLI! This plugin is an early preview, and not ready for production use.
+The CloudFormation CLI (cfn) allows you to author your own resource providers that can be used by CloudFormation.
```

Comment:
This sentence kind of feels incomplete. You can probably add something like,
"...that can be used by CloudFormation to Create/Update/Delete your resource."